capa/helpers.py

```diff
@@ -76,13 +86,46 @@ def load_json_from_path(json_path: Path):
         try:
             report_json = compressed_report.read()
         except gzip.BadGzipFile:
-            report = json.load(json_path.open(encoding="utf-8"))
+            report = msgspec.json.decode(json_path.read_text(encoding="utf-8"))
         else:
-            report = json.loads(report_json)
+            report = msgspec.json.decode(report_json)
     return report
 
 
+def load_jsonl_from_path(jsonl_path: Path) -> Iterator[Dict]:
+    def decode_json_lines(fd: Union[BinaryIO, gzip.GzipFile]):
+        for line in fd:
+            try:
+                line_s = line.strip().decode()
+                obj = msgspec.json.decode(line_s)
+                yield obj
+            except ValueError:
+                # ignore erroneous lines
+                continue
```

Comment:
what are these, should we at least add debug warnings for them?

Reply:
There are often too many lines for each log file, so I've logged that into debug output instead (let me know if there is a better place to log them into).